src/content/about/blog/oscal-first-deep-cleaning.md

```diff
@@ -0,0 +1,24 @@
+---
+title: OSCAL's Fall Cleaning
+---
+
+Welcome to the Open Security Controls Assessment Language (OSCAL) Blog, open to the NIST OSCAL Team and to the community! 
+
+If you work closely with us, you might have noticed we finished [a large code repository reorganization](https://github.com/usnistgov/OSCAL/blob/f24dd56d5569ade8489924cf6fc2640dc297bfbe/decisions/0005-repository-reorganization.md) alongside [the release of OSCAL v1.1.0](https://github.com/usnistgov/OSCAL/releases/tag/v1.1.0). The OSCAL Project was in a desperate need of some internal restructuring, a fall cleaning of sorts. As an OSCAL user, the differences should not affect affect you, but we will still summarize some key changes for you. 
```

Comment:
At the date of the publication, our latest version is OSCAL 1.1.1. I suggest adding the following for accuracy:
" ... alongside [the release of OSCAL v1.1.0](https://github.com/usnistgov/OSCAL/releases/tag/v1.1.0) and the subsequent [OSCAL v1.1.1 patch] (https://github.com/usnistgov/OSCAL/releases/tag/v1.1.1)

Reply:
I thought we were going for concise, but ok, I can add that.

Reply:
https://github.com/usnistgov/OSCAL-Pages/pull/44/commits/1fd136327459d592aabd6b3bc0a4ca6d3dfc92fc